apps/emails/tests/test_tasks.py

```diff
@@ -151,9 +152,9 @@ def test_contribution_confirmation_email_style(
             expect_missing = (custom_logo, custom_button_background, custom_header_background)
 
         else:
-            expect_present = (custom_logo,)
+            expect_present = (custom_logo, custom_header_background)
             # Email template doesn't have a button to apply the custom button color to and also doesn't have a header background to customize
```

Comment:
@nrh-cklimas could you also update the comment?
```suggestion
            # Email template doesn't have a button to apply the custom button color to
```

Reply:
316ea5c1f1de1e273fa01f9b4a5a06cb977c91e1